be/src/olap/rowset/rowset.cpp

```diff
@@ -65,6 +67,9 @@ void Rowset::make_visible(Version version, VersionHash version_hash) {
     _rowset_meta->set_version(version);
     _rowset_meta->set_version_hash(version_hash);
     _rowset_meta->set_rowset_state(VISIBLE);
+    // update create time to the visible time,
+    // it's used to skip recently published version during compaction
+    _rowset_meta->set_creation_time(time(nullptr));
```

Comment:
Better to use function in util/time.h.

Reply:
You mean `UnixMillis() / 1000`? Any reason for that, it seems more verbose than time(nullptr).

Reply:
Just to unify. I think it will be better to use functions in this file to get all kinds of current time.
Actually, it is OK for this case to get a monotonic time other than wall time, which will still work when system time rollback.

Reply:
OK, I'll use functions in util/time.h instead. However I don't think monotonic time is appropriate here for two reasons
1. Previous rowsets use unix timestamp as `creation_time`, switch to monotonic time will break other parts of code that uses that field
2. Monotonic time may go backward after system reboot, which is not suitable for persistence

Reply:
Yes, you're right. Last night when I remembered that create time will be saved, then I realize that the monotonic time is not suitable.